### PR TITLE
LPD-21645 Instance settings default permissions

### PR DESCRIPTION
**Pull request by smottal**
https://liferay.atlassian.net/browse/LPD-21645

### CODE REVIEW

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by CrescenzoRegaLF** [comment]
ci:report:2560828439

File: modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/portlet/action/EditPortalConfigurationPermissionsMVCActionCommand.java [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+		hideDefaultErrorMessage(actionRequest);
+		hideDefaultSuccessMessage(actionRequest);
+
+		try {
+			ThemeDisplay themeDisplay =
+				(ThemeDisplay)actionRequest.getAttribute(WebKeys.THEME_DISPLAY);
+
+			String modelResource = ParamUtil.getString(
+				actionRequest, "modelResource");
+			long[] roleIds = StringUtil.split(
```
Comment (andrea-ale-sbarra) [inline modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/portlet/action/EditPortalConfigurationPermissionsMVCActionCommand.java:69]: inline

File: modules/apps/portal/portal-default-permissions-web/src/main/resources/META-INF/resources/configuration/edit_portal_configuration_permissions.jsp [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+		<clay:button
+			cssClass="btn-cancel"
+			displayType="secondary"
+			label="cancel"
+		/>
+	</aui:button-row>
+</div>
+
+<aui:script>
```
Comment (andrea-ale-sbarra) [inline modules/apps/portal/portal-default-permissions-web/src/main/resources/META-INF/resources/configuration/edit_portal_configuration_permissions.jsp:162]: externalise with module

File: modules/apps/portal/portal-default-permissions-web/src/main/resources/META-INF/resources/configuration/portal_default_permissions_company_configuration.jsp [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+			<liferay-ui:search-iterator
+				displayStyle="list"
+				markupView="lexicon"
+			/>
+		</liferay-ui:search-container>
+	</div>
+</clay:container-fluid>
+
+<aui:script require="frontend-js-web/index as frontendJsWeb">
```
Comment (andrea-ale-sbarra) [inline modules/apps/portal/portal-default-permissions-web/src/main/resources/META-INF/resources/configuration/portal_default_permissions_company_configuration.jsp:49]: externalise

**Review by andrea-ale-sbarra — CHANGES_REQUESTED** [review]
just few minor changes.
Overall looks good.
Thank you!

File: modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/configuration/LayoutPortalDefaultPermissionsAsset.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+		"com.liferay.portal.kernel.model.Layout";
+
+	@Override
+	public String getClassName() {
+		return ASSET_KEY;
+	}
+
+	@Override
+	public String getLabel() {
+		return "Page";
```
[the remark this answers is not in the document]
Comment (smottal) [reply modules/apps/portal/portal-default-permissions-web/src/main/java/com/liferay/portal/defaultpermissions/web/internal/configuration/LayoutPortalDefaultPermissionsAsset.java]: Added the translation in the DisplayContext

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:report:2561297371

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:um

**Comment by smottal** [comment]
ci:test:relevant

**Review by andrea-ale-sbarra — APPROVED** [review]
LGTM.
Thank you!

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:relevant

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:test:sf

**Comment by smottal** [comment]
ci:report:2561586628

**Comment by smottal** [comment]
Send directly here: https://github.com/brianchandotcom/liferay-portal/pull/149392 because the SF errors are not related

**Pull request closed without merging**